kornia/geometry/transform/imgwarp.py

```diff
@@ -438,7 +438,7 @@ def get_rotation_matrix2d(
 
 def remap(tensor: torch.Tensor, map_x: torch.Tensor, map_y: torch.Tensor,
           mode: str = 'bilinear', padding_mode: str = 'zeros',
-          align_corners: Optional[bool] = None) -> torch.Tensor:
+          align_corners: Optional[bool] = None, normalized_coordinates: bool = True) -> torch.Tensor:
```

Comment:
in order to keep backward compatibility I think this parameter should be set by default to `False` since before the method internally was doing the normalization all the time

Reply:
Okay. @edgarriba  I was thinking of renaming it to `normalize_coordinates` in place of `normalized_coordinates`. I misspelled it earlier from the example in the issue. Which one do you think is appropriate?

Reply:
I would keep it as `normalised_coordinates` in order to be consistent with `create_meshgrid` https://github.com/kornia/kornia/blob/master/kornia/utils/grid.py#L10

Reply:
@edgarriba Setting it to `False` as default breaks the other tests because in this `False` means normalization is not required and hence it is not normalized which breaks the previous tests which require normalization. I think the confusion lies in the understanding of the requirement here. As I posted here https://github.com/kornia/kornia/pull/921#discussion_r598951702. There are two assumptions here. Let me know which one to follow.

Reply:
I understand this case as follows, 
`normalised_coordinates` we assume that the input coordinates are not normalise (as we assume right now and that's why should be False by default), next before `grid_sample` we should normalise since assume -1/1 sampling points.

/cc @ducha-aiki @shijianjian @lferraz

Reply:
Umm, in that case, the implementation should be `if **not** normalized_coordinates: normalize it`.
The current implementation is completely opposite to what you mentioned. That makes it clear, I'll make the changes accordingly then.
Please correct me if I'm wrong @edgarriba @ducha-aiki @shijianjian @lferraz